test/evp_kdf_test.c

```diff
@@ -1613,7 +1617,12 @@ static int test_kdf_ss_kmac(void)
 
     ret =
         TEST_ptr(kctx = get_kdfbyname(OSSL_KDF_NAME_SSKDF))
-        && TEST_int_gt(EVP_KDF_derive(kctx, out, sizeof(out), params), 0)
+        && TEST_size_t_eq(EVP_KDF_CTX_get_kdf_size(kctx), 0)
+        && TEST_int_eq(EVP_KDF_CTX_set_params(kctx, params), 1)
+        /* The bug fix for KMAC returning SIZE_MAX was added in 3.1 */
```

Comment:
@slontis As I've marked it for 3.0 should this be changed to 3.0.8 instead? Or it shouldn't be merged to 3.0? IMO this is just a bug fix so I do not see a reason why it shouldn't.

Reply:
Will change to 3.0.8